appcues/src/main/java/com/appcues/ui/composables/AppcuesComposition.kt

```diff
@@ -107,7 +107,10 @@ private fun BoxScope.ComposeLastRenderingState(state: Rendering) {
                         currentPage = state.position,
                         composePage = { index ->
                             with(steps[index]) {
-                                CompositionLocalProvider(LocalAppcuesActions provides actions) {
+                                CompositionLocalProvider(
+                                    LocalAppcuesActions provides actions,
+                                    LocalExperienceStepFormStateDelegate provides formState
```

Comment:
I moved the composition local for the form state here, instead of only within the `ComposeStepContent`. The reason is that it is possible for sticky content to contain form components - really any step decorating traits applied in this spot _might_ have a reason to need access to the form state. I think it was previously too local in the ComposeStepContent.

Reply:
good call